src/platforms/node/guides/aws-lambda/index.mdx

```diff
@@ -16,37 +19,60 @@ $ npm install --save @sentry/node
 $ yarn add @sentry/node
 ```
 
-To set up Sentry error logging for a Lambda Function, build a wrapper:
+You can use the AWS Lambda integration for the Node SDK like this:
 
 ```javascript
-"use strict";
+const Sentry = require("@sentry/serverless");
+
+Sentry.init({
+  dsn:
+    "___PUBLIC_DSN___",
+});
+
+const myAsyncHandler = async (event, context, callback) => {
```

Comment:
```suggestion
const myAsyncHandler = async (event, context) => {
```

`async` functions are mutually exclusive with `callback`. It's either async or sync with callback.